SignalServiceKit/src/Contacts/SignalAccount.m

```diff
@@ -46,6 +47,23 @@ - (nullable SignalRecipient *)signalRecipientWithTransaction:(YapDatabaseReadTra
     return [SignalRecipient recipientWithTextSecureIdentifier:self.recipientId withTransaction:transaction];
 }
 
+- (nullable NSString *)uniqueId
+{
+    return _recipientId;
+}
+
+- (NSString *)displayName
+{
+    NSString *baseName = (self.contact.fullName.length > 0 ? self.contact.fullName : self.recipientId);
+
+    OWSAssert(self.hasMultipleAccountContact == (self.multipleAccountLabelText != nil));
+    NSString *displayName = (self.multipleAccountLabelText
```

Comment:
This isn't redundant with logic in the `OWSContactsManager` is it?

Reply:
This logic was moved from OWSContactsManager, but it lives only here now.